### PR TITLE
Included a model for the server

### PR DESCRIPTION
**Pull request by briansai**

### CODE REVIEW

**Comment by briansai** [comment]
@lighthurst  It's under server/model.js.  Thanks!

File: server/model/model.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        FROM description_to_amenities da 
+        INNER JOIN amenities a 
+          ON da.amenities_id = a.id) AS b
+        ON d.id = b.description_id
+        LEFT JOIN (SELECT dhr.description_id, hr.house_rules
+        FROM description_to_house_rules dhr
+        INNER JOIN house_rules hr
+          ON dhr.house_rules_id = hr.id) AS c
+        ON d.id = c.description_id  
+        WHERE d.id = ?;`;
```
Comment (lighthurst) [inline server/model/model.js:18]: Wow, 13 lines for this query. God help you man. I won't even pretend to know the amount of work that went in to get the accurate data you need here. As long as it works, looks good.

File: server/model/model.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    db.con.query(q, parameter, (err, result) => {
+      if (err) {
+        callback(err);
+      } else {
+        callback(result);
```
Comment (lighthurst) [inline server/model/model.js]: Ensure proper callback arguments. if (err) callback(err) else callback(null, result);

**Review by lighthurst — CHANGES_REQUESTED** [review]
Nice work Brian. You are the MySQL master.

File: server/model/model.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ module.exports = {
       if (err) {
         callback(err);
       } else {
-        callback(result);
+        callback(null, result);
```
Comment (briansai) [inline server/model/model.js]: @lighthurst  I have included proper callback in this section. Thanks!

**Review by lighthurst — APPROVED** [review]

**Pull request merged**